rosdoc.yaml

```diff
@@ -1 +1,2 @@
- - builder: epydoc
+ - builder: sphinx
```

Comment:
Switching from epydoc, which has not been maintained (last release was in 2008 seeing from [its website](http://epydoc.sourceforge.net/)) AFAIK, while `Sphinx` is actively maintained.